### ISSUE TITLE
Feature req: Natively support mouse gestures & chording

### ISSUE THREAD

**Issue by rodalpho** [brave/brave-browser#6014]
Neither Firefox nor Chrome support mouse gestures natively. Even after many years, WebExtensions still don't work well. They load after the DOM so you can't use gestures while loading, they don't work on internal pages (particularly including new tab), performance is laggy/inconsistent, and really they just offer an unpleasant experience.

Vivaldi does offer native gestures and they work great, however Vivaldi does not permit users to customize mouse _chording and wheel gestures._ Chording is when you hold down one mouse button then click the other one. Wheel gestures are when you hold down the RMB and then scroll the mouse wheel.

Opera doesn't permit the user to customize gestures at all and is thus useless to me.

This feature request is for Brave to support both gestures and chording natively and just, basically offer an experience that doesn't suck like Firefox and Chrome.

**Comment by ellman121** [comment]
Strong agree, would very much love to see this brought in.  Would make the migration from Vivaldi much less painful

**Comment by recalcitrant** [comment]
This is the only thing preventing me from moving from Vivaldi to Brave. It could be so easy...

**Comment by rodalpho** [comment]
Yep, I'm sticking with Firefox myself and dealing with all the petty annoyances because I use chording and wheel gestures too.
